app/validators/base_validator.rb

```diff
@@ -17,6 +17,7 @@ def validate(record)
 
   def validate_fields(fields_class_method)
     return unless self.class.respond_to?(fields_class_method)
+
```

Comment:
Remove empty line?

Reply:
good to have a line break after a return statement, for clarity

Reply:
👍